pkg/operator/deploy/staticresources/aro.openshift.io_clusters.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.5.0
+    controller-gen.kubebuilder.io/version: v0.6.2
```

Comment:
sorry for my ignorance, where does this version come from?

Reply:
@bryanro92 Good question. I should've explained it too as it is not obvious.

We generate this yaml using `controller-gen` which is part of `sigs.k8s.io/controller-tools`. It got updated from `v0.5.0` to `v0.6.2` and as a result on `make generate` this yaml got updated too.

You can see version change of `sigs.k8s.io/controller-tools` in `go.mod`.